uber/tests/email/email_fixtures.py

```diff
@@ -0,0 +1,165 @@
+from uber.tests import *
+from unittest.mock import patch, Mock
+
+
+class EmailTestsConstants:
+    SUBJECT_TO_FIND = 'CoolCon9000: You Need To Know'
+    IDENT_TO_FIND = 'you_are_not_him'
+
+E = EmailTestsConstants
+
+
+@pytest.fixture
+def remove_all_email_categories(monkeypatch):
+    monkeypatch.setattr(AutomatedEmail, 'instances', OrderedDict())
+
+
+@pytest.fixture
+def add_test_email_categories(remove_all_email_categories):
```

Comment:
I think this might be ok as-is because it requires the 'remove_all_email_categories' fixture to be applied before each time this fixture is invoked, which resets AutomatedEmail.instances to an empty dict.

I did verify that each of these fixtures is getting re-applied before every single test runs, so it shouldn't be possible, even if we called this a couple times, for us to end up with duplicate results here